lib/generator/test_template.erb

```diff
@@ -6,5 +6,5 @@ class <%= exercise_test_classname %> < Minitest::Test
 <%=
   test_cases.map.with_index do |test_case, index|
     test_case.to_s(index.zero?)
-  end.join("\n")
+  end.compact.join("\n")
```

Comment:
Without this change the resulting `acronym_test.rb` would end with:

```ruby
  end


end
```

Instead of:

```ruby
  end
end
```

Reply:
😍